### ISSUE TITLE
compression Error for wr.s3.to_json

### ISSUE THREAD

**Issue by Mzhussiyev** [aws/aws-sdk-pandas#1582]
### Describe the bug

When I tried to pass the value to **compression** parameter for s3.to_json method I faced with an error - "_awswrangler.s3._write_dataset._to_dataset() got multiple values for keyword argument 'compression'_".
E.g. The same parameters works with to_csv method but not works with to_json.

### How to Reproduce

```
data = [10,20,30,40,50,60]
df = pd.DataFrame(data, columns=['Numbers'])

params = {
        "df": df,
        "path": S3_PATH,
        "dataset": True,
        "mode": "overwrite" ,
        "compression": "gzip",
        "index": False,
        "filename_prefix": "test_file",
        "boto3_session": session,
    }

res = wr.s3.to_json(**params)
print(res)
```

### Expected behavior

As expectation it should create a compressed file like (test_file.json.gz)

### Screenshots

![image](https://user-images.githubusercontent.com/73746305/188947484-2d2a80e6-f724-4725-936e-d43cea2e63da.png)

### OS

WSL2

### Python version

3.9.10

### AWS SDK for pandas version

2.16.1

**Comment by Mzhussiyev** [comment]
@jaidisido thanks a lot. Is it already  merged to master? And could you tell me please when these changes will be available in PyPi?

**Comment by jaidisido** [comment]
Yes it's merged to the main branch and you should be able to pip install the fix from there. Otherwise it will be available in the next release (2.17.0) in the next few days
